ptf/saiport.py

```diff
@@ -0,0 +1,2829 @@
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=True)
+            self.assertEqual(attr['qos_pfc_priority_to_priority_group_map'],
+                             initial_qos_map_id)
+
+            if TEST_QOS_MAP_ON_CREATE_PORT is True:
+                status = sai_thrift_set_port_attribute(
+                    self.client, self.portx,
+                    qos_pfc_priority_to_priority_group_map=0)
+                self.assertEqual(status, SAI_STATUS_SUCCESS,
+                                 "Failed to set port attribute")
+
+            # set/get
+            status = sai_thrift_set_port_attribute(
+                self.client,
+                self.portx,
+                qos_pfc_priority_to_priority_group_map=qos_map_id)
+            self.assertEqual(status, SAI_STATUS_SUCCESS,
+                             "Failed to set port attribute")
+
+            attr = sai_thrift_get_port_attribute(
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=True)
+            self.assertEqual(attr['qos_pfc_priority_to_priority_group_map'],
+                             qos_map_id)
+
+        finally:
+            # Remove the qos_map
+            status = sai_thrift_set_port_attribute(
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=0)
+            status = sai_thrift_remove_qos_map(self.client, qos_map_id)
+            status = sai_thrift_remove_port(self.client, self.portx)
+
+
+@group("draft")
+class SAIBridgePortNoFloodTest(SaiHelperBase):
+    ''' Verifies SAI bridge port no flood test case '''
+
+    def setUp(self):
+
+        super(SAIBridgePortNoFloodTest, self).setUp()
+
+        self.test_port_list = [self.port0, self.port1, self.port2, self.port3]
+        self.dev_test_port_list = [
+            self.dev_port0,
+            self.dev_port1,
+            self.dev_port2,
+            self.dev_port3]
+        vlan_id = 1
+        for port in self.test_port_list:
+            attr = sai_thrift_set_port_attribute(self.client, port,
+                                                 port_vlan_id=vlan_id)
+            attr = sai_thrift_get_port_attribute(self.client, port,
+                                                 port_vlan_id=True)
+            self.assertTrue(attr['SAI_PORT_ATTR_PORT_VLAN_ID'] == vlan_id)
+
+    def runTest(self):
+        print("SAIBridgePortNoFloodTest")
+
+        src_mac = "00:11:11:11:11:11"
+        dst_mac = "00:22:22:22:22:22"
+
+        pkt = simple_udp_packet(eth_dst=dst_mac,
+                                eth_src=src_mac,
+                                pktlen=100)
+        print("Sending packet on port with no bridge port created")
+        send_packet(self, self.dev_port1, pkt)
+        verify_no_other_packets(self)
+        print("\tPacket dropped. OK")
+
+
+@group("draft")
+class SAIBridgePortFloodClass(SaiHelperBase):
```

Comment:
same here

Reply:
SAIBridgePortFloodClass is used as a base class for PortIngressMirrorSessionTest, PortEgressMirrorSessionTest and PortAclBindingClass, I renamed it to PortFloodClass